src/mixins/meta.ts

```diff
@@ -127,3 +161,44 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+/**
+ * Alternate locales meta and link tags generator.
+ * @param locales Array of alternate locales the the webpage is currently available in.
+ * @param pageCurrentLocale
+ * @returns 
+ */
+function pageAlternateLocales(locales: QMetaI18nAlternateLocale[], pageCurrentLocale?: QMetaI18nAlternateLocale) {
```

Comment:
`pageCurrentLocale` becomes `currentLocale` and is provided by `this.$root.$i18n.locale`, not optional anymore